wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandRobot.java

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj2.command;
+
+import edu.wpi.first.wpilibj.TimedRobot;
+
+/**
+ * CommandRobot is a wrapper over TimedRobot designed to work well with Command based programming.
+ *
+ * <p>The CommandRobot class is intended to be subclassed by a user creating a robot program.
+ */
+public class CommandRobot extends TimedRobot {
+  /** The CommandScheduler instance. */
+  protected CommandScheduler m_scheduler = CommandScheduler.getInstance();
+
+  /** The Command run during autonomous. */
+  protected Command m_autonomousCommand =
```

Comment:
This doesn't really feel right to me. If you're going to add support for running an autonomous command directly, seems like it should be easy to integrate with a SendableChooser (or use one by default).

Telling the user to set a variable feels like they need to know too much about how the class works internally which may cause problems in the future.

Reply:
I'd be happy to make it use a SendableChooser by default

Reply:
I'd probably lean towards leaving this out completely, but I don't use commands so I'm not sure I'm the right person to make that call.

Reply:
We either have it as a member variable or add the getAutoCommand
Method back